features/org.wso2.carbon.core.common.feature/pom.xml

```diff
@@ -170,6 +168,11 @@
                                     org.wso2.carbon.crypto:org.wso2.carbon.crypto.provider:${carbon.crypto.version}
                                 </bundleDef>
                                 <bundleDef>org.wso2.orbit.sun.xml.bind:jaxb:${version.org.wso2.orbit.javax.xml.bind.jaxb}</bundleDef>
+                                <!-- TODO previously javax.activation package has come from org.eclipse.osgi_3.14.0.v20190517-1309 -->
+                                <bundleDef>org.wso2.orbit.javax.activation:activation:1.1.1.wso2v4</bundleDef>
+                                <!-- TODO previously javax.transaction package has come from org.eclipse.osgi_3.14.0.v20190517-1309 -->
+                                <!-- TODO previously javax.xml.soap was provided by org.eclipse.osgi_3.14.0.v20190517-1309 -->
+                                <bundleDef>javax.xml.soap:javax.xml.soap-api:1.4.0</bundleDef>
```

Comment:
Check if this is the best component to add javax.xml.soap. Based on this, update the import packages as well.